### PR TITLE
REF: BlueCopyToClipboardButton and BlueCopyToClipboard to TSX

### PR DESCRIPTION
**Pull request by marcosrdz**

### CODE REVIEW

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/aidci3yyzminfvzmxxzwl7c4a4?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/966)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/6qthxbp7j7xy2p2dgvbyecuylq?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/969)

**Comment by GladosBlueWallet** [comment]
Wake the fuck up samurai, we have PRs to merge

image

[all PRs for @limpbrains] https://github.com/BlueWallet/BlueWallet/pulls/review-requested/limpbrains

**Comment by GladosBlueWallet** [comment]
Wake the fuck up samurai, we have PRs to merge

image

[all PRs for @Overtorment] https://github.com/BlueWallet/BlueWallet/pulls/review-requested/Overtorment

**Review by limpbrains — APPROVED** [review]

File: components/CopyTextToClipboard.tsx [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+  useEffect(() => {
+    if (!hasTappedText) {
+      setAddress(text);
+    }
+  }, [text, hasTappedText]);
+
+  const copyToClipboard = () => {
+    setHasTappedText(true);
+    Clipboard.setString(text);
+    setAddress(loc.wallets.xpub_copiedToClipboard); // Adjust according to your localization logic
```
Comment (Overtorment) [inline components/CopyTextToClipboard.tsx:33]: how do i adjust it? its hardcoded

**Review by Overtorment — APPROVED** [review]

**Comment by GladosBlueWallet** [comment]
Unbelievable. You, [subject name here], must be the pride of [subject hometown here]!

**Pull request merged**